src/client.cpp

```diff
@@ -122,7 +121,7 @@ CClient::CClient ( const quint16  iPortNumber,
     QObject::connect ( &Channel, &CChannel::ConClientListMesReceived, this, &CClient::OnConClientListMesReceived );
     QObject::connect ( &Channel, &CChannel::ConClientListMesReceived, this, &CClient::ConClientListMesReceived );
 
-    QObject::connect ( &Channel, &CChannel::Disconnected, this, &CClient::Disconnected );
+    QObject::connect ( &Channel, &CChannel::Disconnected, this, &CClient::Stop );
```

Comment:
If this _only_ gets used _as a slot_ by `CChannel::Disconnected` could we use a name like `OnChannelDisconnected` rather than the somewhat ambiguous `Stop`?

Reply:
Oh, I see it's also used just before exiting the client in the `SIGINT`/`SIGTERM` handler...  Hm.

`SIGINT`/`SIGTERM` and "File -> Exit (Ctrl-Q)" should both really do the same thing...

I'd rather we had a lambda here, `(whatever) -> Stop()`, so it was clear that `Stop` wasn't a slot but a normal method.